pop/POPAnimatableProperty.mm

```diff
@@ -568,7 +569,25 @@
     },
     0.01
   },
-
+  
+  {kPOPLayoutConstraintMultiplier,
+    ^(NSLayoutConstraint *obj, CGFloat values[]) {
+      values[0] = obj.multiplier;
+    },
+    ^(NSLayoutConstraint *obj, const CGFloat values[]) {
+//  The multiplier property is read-only so recreate the constraint with all existing attributes
+//  besides the desired multiplier.
+      obj = [NSLayoutConstraint constraintWithItem:obj.firstItem
```

Comment:
Hm, I'm not sure if this will work. Wouldn't you need to also update / replace the constraint on the view it's attached to?